.github/workflows/hosted-post-merge_version_update.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: enable toolchain via github action
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: 1.54
+          toolchain: 1.58.0
```

Comment:
Here specifies one more digit than before. I guess the reason, in our commit workflow we specified 3 digit like `1.54.0`, and keep here and the commit workflow consistent?